example/spec/redeployment/LambdaRedeploySpec.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const {
+  buildAndStartWorkflow,
+  waitForCompletedExecution,
+  LambdaStep
+} = require('@cumulus/integration-tests');
+
+const {
+  loadConfig,
+  redeploy
+} = require('../helpers/testUtils');
+
+const { updateConfigObject } = require('../helpers/configUtils');
+
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+
+describe('When a workflow', () => {
```

Comment:
since the tests aren't running yet, you might xdescribe the whole suite so that the beforeAll()s don't run.  But now that I'm typing that, maybe they won't since there aren't any tests to run before.

Reply:
I believe they don't run unless there's a test.